### ISSUE TITLE
URGENT: Use HTTPS endpoint for AWIS and Alexa TopSites API

### ISSUE THREAD

**Issue by eladnava** [wrangr/awis#11]
Hi, 
I have just received the following e-mail from Amazon Web Services:

![screen shot 2017-08-12 at 11 46 37 am](https://user-images.githubusercontent.com/4394189/29239283-e9b5782c-7f53-11e7-8806-a0a26a5d8bf4.png)

I have noticed that this package uses the `http` endpoint:

```js
    Request({
      url: 'http://' + apiDomain,
      qs: internals.query(req, apiDomain, options)
    }, (err, res) => {
```

[index.js:L171](https://github.com/wrangr/awis/blob/5b186668f7cdc88f852f4447eda035259a696ae2/index.js#L170-L173)

Can I submit a PR to update the package to use the `https` endpoint?  @lupomontero

**Comment by CelsoSantos** [comment]
I came here to say and ask precisely the same lol

**Comment by eladnava** [comment]
@CelsoSantos Since the migration is effective in 6 days, and there has not been any response from @wrangr, I have published `awis-https` after modifying this package to connect via `https`:
https://www.npmjs.com/package/awis-https

Consider using this package until the change becomes effective in this package. 👍

**Comment by lupomontero** [comment]
Hi @CelsoSantos and @eladnava, thanks for the heads up and apologies for the delay in getting back to you guys. I have now merged the [relevant PR](https://github.com/wrangr/awis/pull/12) and published `v1.0.9`.

Thanks!
